pkg/cmd/upgradecmd.go

```diff
@@ -476,6 +485,8 @@ func getUpgradeMethod(fileSystem vfs.Stater, executableAbsPath chezmoi.AbsPath)
 		return upgradeMethodBrewUpgrade, nil
 	case runtime.GOOS == "linux" && strings.Contains(executableAbsPath.String(), "/.linuxbrew/"):
 		return upgradeMethodBrewUpgrade, nil
+	case runtime.GOOS == "windows" && strings.Contains(executableAbsPath.String(), "/WinGet/"):
```

Comment:
This might need a case-insensitive match.

Reply:
It's matching `C:\\<username>\\AppData\\Local\\Microsoft\\WinGet\\Links\\chezmoi.exe`.

I just realised it doesn't actually match with `/WinGet/`, it matches with `\\WinGet\\`.